### ISSUE TITLE
W-8BEN and W-8BEN-E forms 

### ISSUE THREAD

**Issue by anujtenani** [anujtenani/goaffpro#131]
Allow for filling of W-8BEN form by non-us individuals and W-8BEN-E by non-us businesses in the W9 form functionality

**Comment by worldtravlr** [comment]
Yes!  Love the way the W9 form works, but we are unable to use it until W-8BEN and W-8BEN-E are added.
